prow/cmd/deck/static/prow/prow.ts

```diff
@@ -650,7 +650,13 @@ function redraw(fz: FuzzySearch): void {
         max = 2 * 3600;
     }
     drawJobHistogram(totalJob, jobHistogram, now - (12 * 3600), now, max);
-    if (rerunStatus != null) {
+    if (rerunStatus === "denied") {
+        modal.style.display = "block";
+        rerunCommand.innerHTML = `You don't have permission to rerun that job. Try asking @cjwagner.`;
+    } else if (rerunStatus === "success") {
+        modal.style.display = "block";
+        rerunCommand.innerHTML = `Job successfully triggered`;
```

Comment:
We should probably tell users to wait 30 seconds or so and then refresh the page to see the new job. Otherwise they might rerun again if they don't think the job actually started.

Eventually it would also be nice to automatically update the filters to show the new job or provide a link to such filters if automatically changing the filters would be annoying.